### DISCUSSION TITLE
how to export the lsf data?

### DISCUSSION THREAD

**Discussion by zxylyi** [JorgeGIlG/MTF_Estimator#39] [Q&A]
Hi i have tried your program in QGIS, that is powerful. But the results are shown as image. Do you know any way to export the result as detailed data? I want to analyze the lsf data, thanks!

**Comment by JorgeGIlG** [comment]
Hi.
Everything in that figure is plotted [here](https://github.com/JorgeGIlG/MTF_Estimator/blob/0cbf272aa24af968efb8b0172dfd87a5c901ace6/qgis_plugin/mtf_estimator/mtf_estimator_algorithm.py#L382). You can export whatever you need to disk with using Numpy's savetxt method, for example.
`np.savetxt('lsf.csv', [x,y], delimiter=',')`
It is a nice feature. It would be great if you could contribute some sort of export function to the results window.
